desktop/apps/editorial_features/components/venice_2017/templates/index.jade

```diff
@@ -3,6 +3,7 @@ extends ../../../../../components/main_layout/templates/blank.jade
 append locals
   - assetPackage = 'editorial_features'
   - section = curation.get('sections')[videoIndex]
+  - options = {modal: false, stripe: false, sailthru: false, marketo: false}
```

Comment:
Ah we'll want ST here, we send metadata to them in the `head` template and this will let us track pvs there.